authors/apps/authentication/serializers.py

```diff
@@ -15,10 +15,11 @@ class RegistrationSerializer(serializers.ModelSerializer):
         max_length=128,
         min_length=8,
         write_only=True,
-        required = True,
-        validators =[RegexValidator(
-            regex='^[a-zA-Z0-9]*',
-            message = 'Password should be alphanumeric',
+        required=True,
+        validators=[RegexValidator(
+            regex="^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$",
+            message="Please ensure your password contains at least one letter\
```

Comment:
The backslash in the message causes an abnormal number of whitespaces to be inserted in the response.

Reply:
Also line 26 still contains the message 'min_length': 'Password should be at least 8 characters longclear',

Reply:
> Also line 26 still contains the message 'min_length': 'Password should be at least 8 characters longclear',

This has been fixed in another PR. `bg-fix-typo`

Reply:
> The backslash in the message causes an abnormal number of whitespaces to be inserted in the response.

fixing it right a way